### ISSUE TITLE
`needs-go-generate` labels are not removed

### ISSUE THREAD

**Issue by ChiTimesChi** [synapsecns/sanguine#1397]
An old PR could get a bunch of `needs-go-generate` labels if it is behind master, see for example:

https://github.com/synapsecns/sanguine/actions/runs/6265362203/job/17047824402

However, once the `master` branch is merged into the branch in question, the labels persist. This is due to the whole `Go Generate (Module Changes)` workflow being skipped on no changes:
https://github.com/synapsecns/sanguine/actions/runs/6265608504/job/17048082573

The desired behavior would be: when no changes are detected for a module, the `needs-go-generate` label for the module should be removed (if there is one).

**Comment by trajan0x** [comment]
Fixed in #1399
